Fluid/Values/FluidValue.cs

```diff
@@ -51,6 +51,11 @@ public virtual bool IsNil()
         }
 
         public static FluidValue Create(object value)
+        {
+            return Create(value, true);
+        }
+
+        public static FluidValue Create(object value, bool encode)
```

Comment:
We can make it an optional parameter
```csharp
public static FluidValue Create(object value, bool encode = false)
```

Reply:
Since signature Create(object value) has been used in several Lambda expressions as a type, making it to have optional parameter will force us to refactor all the cases. I think overriding the method is simpler solution and wouldn't break any usage